src/System Application/Test/EntityText/src/MktTextAccuracyAIT.Codeunit.al

```diff
@@ -0,0 +1,240 @@
+        Format: Enum "Entity Text Format";
+        Category: Text;
+        InputPrompt: Text;
+        Response: Text;
+    begin
+        InputPrompt := AITestContext.GetInput().ValueAsText();
+        Facts := LineInputToDictionary(InputPrompt);
+        Format := Enum::"Entity Text Format"::Tagline;
+        Response := EntityTextCod.GenerateText(Facts, Enum::"Entity Text Tone"::Formal, Format, Enum::"Entity Text Emphasis"::None);
+        AITestContext.SetTestOutput(PrepareOutput(BuildFactsText(Facts, Category), Category, Response, Enum::"Entity Text Tone"::Formal, Format));
+    end;
+
+    [Test]
+    procedure TaglineCreative()
+    var
+        AITestContext: Codeunit "AIT Test Context";
+        EntityTextCod: Codeunit "Entity Text";
+        Facts: Dictionary of [Text, Text];
+        Format: Enum "Entity Text Format";
+        Category: Text;
+        InputPrompt: Text;
+        Response: Text;
+    begin
+        InputPrompt := AITestContext.GetInput().ValueAsText();
+        Facts := LineInputToDictionary(InputPrompt);
+        Format := Enum::"Entity Text Format"::Tagline;
+        Response := EntityTextCod.GenerateText(Facts, Enum::"Entity Text Tone"::Creative, Format, Enum::"Entity Text Emphasis"::None);
+        AITestContext.SetTestOutput(PrepareOutput(BuildFactsText(Facts, Category), Category, Response, Enum::"Entity Text Tone"::Creative, Format));
+    end;
+
+    [Test]
+    procedure BriefInspiring()
+    var
+        AITestContext: Codeunit "AIT Test Context";
+        EntityTextCod: Codeunit "Entity Text";
+        Facts: Dictionary of [Text, Text];
+        Format: Enum "Entity Text Format";
+        Category: Text;
+        InputPrompt: Text;
+        Response: Text;
+    begin
+        InputPrompt := AITestContext.GetInput().ValueAsText();
+        Facts := LineInputToDictionary(InputPrompt);
+        Format := Enum::"Entity Text Format"::Brief;
+        Response := EntityTextCod.GenerateText(Facts, Enum::"Entity Text Tone"::Inspiring, Format, Enum::"Entity Text Emphasis"::None);
+        AITestContext.SetTestOutput(PrepareOutput(BuildFactsText(Facts, Category), Category, Response, Enum::"Entity Text Tone"::Inspiring, Format));
+    end;
+
+    [Test]
+    procedure BriefFormal()
+    var
+        AITestContext: Codeunit "AIT Test Context";
+        EntityTextCod: Codeunit "Entity Text";
+        Facts: Dictionary of [Text, Text];
+        Format: Enum "Entity Text Format";
+        Category: Text;
+        InputPrompt: Text;
+        Response: Text;
+    begin
+        InputPrompt := AITestContext.GetInput().ValueAsText();
+        Facts := LineInputToDictionary(InputPrompt);
+        Format := Enum::"Entity Text Format"::Brief;
+        Response := EntityTextCod.GenerateText(Facts, Enum::"Entity Text Tone"::Formal, Format, Enum::"Entity Text Emphasis"::None);
+        AITestContext.SetTestOutput(PrepareOutput(BuildFactsText(Facts, Category), Category, Response, Enum::"Entity Text Tone"::Formal, Format));
```

Comment:
This code is repeated in all procedures, and its difference is minimal:
![image](https://github.com/microsoft/BCApps/assets/59221826/65473b8d-b683-475f-9ed2-4db03832d069)

Can you just make one local procedure with parameter for Enum::"Entity Text Format" and parameter for Enum::"Entity Text Tone"

Than you won't have 5 times declaration of 7 variables and use only one liners